packages/@tinacms/toolkit/src/packages/fields/plugins/GroupListFieldPlugin.tsx

```diff
@@ -83,29 +84,40 @@ const Group = ({ tinaForm, form, field, input }: GroupProps) => {
     [field.itemProps]
   )
 
+  // @ts-ignore
+  const isMax = items.length >= (field.max || Infinity)
```

Comment:
I think this is what we'd want, maybe?
```suggestion
  const isMax = items.length > (field.max || Infinity)
```

Reply:
If it were `>` a list with 3 items would be considered valid given a max of 2, so we need `>=`.

Reply:
`isMax` is used to determine if the add button should be disabled. It should be disabled when the list is as many items long as the max or more.

Reply:
Here's a field with `min: 1, max: 3,` set. It has 3 items, so the add button is disabled but the delete buttons aren't.

<img width="401" alt="Screenshot 2023-01-11 at 1 38 14 PM" src="https://user-images.githubusercontent.com/5075484/211878264-8092fd57-345d-4b13-9266-69c32cda8736.png">

Reply:
ahh, I was thinking that bool was used to show the error message. If it's only used for the button-active-state these comments wouldn't apply